.aegir.js

```diff
@@ -0,0 +1,26 @@
+export default {
+  tsRepo: false,
+  build: {
+    config: {
+      external: ['fs'],
+      format: 'esm',
+      banner: {
+        js: ''
+      },
+      footer: {
+        js: ''
+      }
+    }
+  },
+  test: {
+    before: (...args) => {
+      if (args[0].runner === 'node') {
+        return {
+          env: {
+            NODE_OPTIONS: '--loader=esmock'
```

Comment:
Adding this to accommodate: https://github.com/iambumblehead/esmock#:~:text=esmock%20is%20used%20with%20node%27s%20%2D%2Dloader